participants/yitik-kawatra.md

```diff
@@ -1,5 +1,15 @@
-Name: Yitik Kawatra
-GitHub username: yitik-kawatra
-University: GGSIPU
-Year: 2nd
-Branch: ECE
+<h1>Name: Yitik Kawatra</h1>
+<h2>GitHub username:[yitik-kawatra](https://https://github.com/yitik-kawatra/
+</h2>
+<div>
+<h3>
```

Comment:
Don't you think having an entire list inside an `<h3>` tag makes the list bold for no reason?